app/survey_config/version.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+
+class Version(Enum):
+    V2 = "v2"
```

Comment:
Maybe `AuthPayloadVersion`?. Version on its own is too vague.

Reply:
Renamed and moved this 👍